src/devtools/CloseButton.vue

```diff
@@ -1,5 +1,5 @@
 <script lang="ts">
-import { computed, defineComponent, PropType } from "vue";
+import { computed, defineComponent, PropType } from "vue-demi";
```

Comment:
Since dev-tools will not be compatible with Vue 2 anyway for now, you can leave the imports as-is